velox/functions/prestosql/Probability.h

```diff
@@ -118,6 +119,26 @@ struct CauchyCDFFunction {
   }
 };
 
+template <typename T>
+struct LaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double x) {
+    result = std::numeric_limits<double>::quiet_NaN();
+    if (std::isnan(location) || std::isnan(scale) || std::isnan(x)) {
+    } else if (scale <= 0) {
```

Comment:
Its already determined that scale <= 0, then why the check inside the condition for scale > 0 ? Its always going to error then.